lib/main.js

```diff
@@ -404,12 +410,13 @@ proto.hooks = {
             var songInfo = that.playback.getSongInfo();
             // Make sure that this is the first of the notifications for the
             // insertion of the song information elements.
-            if (lastTitle !== songInfo.title || lastArtist !== songInfo.artist || lastAlbum !== songInfo.album) {
+            if (lastTitle !== songInfo.title || lastArtist !== songInfo.artist || lastAlbum !== songInfo.album || lastArt !== songInfo.art) {
```

Comment:
This line is too long and causing linting to fail. Can you break it into multiple lines?

```js
if (lastTitle !== songInfo.title || lastArtist !== songInfo.artist ||
    lastAlbum !== songInfo.album || lastArt !== songInfo.art) {
  that.emit('change:song', songInfo);
```